sklearn/ensemble/tests/test_voting.py

```diff
@@ -503,12 +513,15 @@ def test_transform():
 )
 @pytest.mark.parametrize("drop", [None, 'drop'])
 def test_none_estimator_with_weights(X, y, voter, drop):
-    # check that an estimator can be set to None and passing some weight
+    # check that an estimator can be set to 'drop' and passing some weight
     # regression test for
     # https://github.com/scikit-learn/scikit-learn/issues/13777
+    voter = clone(voter)
```

Comment:
It's interesting to see this pop up because `Voting*.fit` does not clone.

Reply:
This is not even an issue with cloning. We change the `drop` attribute of the `voter` object. `fit` would only reinitialize the underscore attribute.